src/modules/freia/FreiaInstrument.cpp

```diff
@@ -84,15 +84,16 @@ void FreiaInstrument::loadConfigAndCalib() {
 }
 
 void FreiaInstrument::processReadouts(void) {
+  XTRACE(DATA, DEB,"\n================== NEW PACKET =====================\n\n");
   // All readouts are potentially now valid, but rings and fens
   // could still be outside the configured range, also
   // illegal time intervals can be detected here
   assert(Serializer != nullptr);
   Serializer->checkAndSetReferenceTime(
-      ESSReadoutParser.Packet.Time
-          .TimeInNS); /// \todo sometimes PrevPulseTime maybe?
+      /// \todo sometimes PrevPulseTime maybe?
+      ESSReadoutParser.Packet.Time.TimeInNS);
 
-  XTRACE(DATA, DEB, "processReadouts()");
+  //XTRACE(DATA, DEB, "processReadouts()");
```

Comment:
Same as in my previous review. XTRACE is optional by default if we enable debugging. Is there any advantage that you comment out instead of delete or keep it?

Reply:
Too verbode, will remove